docs/stakers/co_staking.mdx

```diff
@@ -0,0 +1,402 @@
+
+<ThemedImage
+  alt="Review and Confirm"
+  sources={{
+    light: useBaseUrl('/img/guides/co-staking/review_confirm.png'),
+    dark: useBaseUrl('/img/guides/co-staking/review_confirm_dark.png'),
+  }}
+/>
+
+4. **Verify Your Delegation**
+- Check that your BABY delegation is active in the dashboard
+
+<ThemedImage
+  alt="Review and Confirm"
+  sources={{
+    light: useBaseUrl('/img/guides/co-staking/baby_activity.png'),
+    dark: useBaseUrl('/img/guides/co-staking/baby_activity_dark.png'),
+  }}
+/>
+
+
+---
+
+### Step 3: Understand Co-Staking Rewards
+
+Co-staking rewards are calculated based on a **weighted formula** that considers both your BTC and BABY stakes.
+
+#### The Formula
+
+For each co-staker, the weight is calculated as:
+
+```
+w = min(B_BABY / R, B_BTC)
+```
+
+Where:
+- `B_BABY` = Amount of BABY you have staked
+- `B_BTC` = Amount of BTC you have staked to active FPs
+- `R` = Co-staking factor = 20,000
+
+This means: **Every 20,000 BABY staked makes 1 BTC eligible for co-staking rewards.**
+
+**Important:** The weight can be any decimal value - you don't need at least 1 BTC or 20,000 BABY to earn rewards! Even small amounts will earn proportional rewards.
+
+#### Optimal Ratio Visualization
+
+The best ratio for maximizing co-staking rewards is **20,000 BABY per 1 BTC**:
+
+| BTC Staked | Optimal BABY | Co-Staking Weight | Efficiency |
+|------------|--------------|-------------------|------------|
+| 0.1 BTC | 2,000 BABY | 0.1 BTC-equivalent | 100% ✅ |
+| 0.5 BTC | 10,000 BABY | 0.5 BTC-equivalent | 100% ✅ |
+| 1 BTC | 20,000 BABY | 1 BTC-equivalent | 100% ✅ |
+| 1 BTC | 10,000 BABY | 0.5 BTC-equivalent | 50% ⚠️ |
+| 1 BTC | 40,000 BABY | 1 BTC-equivalent | 100% ✅ (excess BABY) |
+| 6 BTC | 120,000 BABY | 6 BTC-equivalent | 100% ✅ |
+
+#### Reward Distribution
+
+Your co-staking reward is calculated as:
+
+```
+Your Reward = Total Co-Staking Pool × (Your Weight / Total Weight of All Co-Stakers)
+```
+
+The total co-staking pool is **2.35% of annual inflation**.
+
+---
+
+
+### Step 4: Monitor Your Rewards
+
+1. **Check Your Co-Staking Status**
+- Verify that both your BTC and BABY stakes are active
+
+2. **Calculate Your Expected Rewards**
+- 👉 **[Use the Co-Staking Calculator](#co-staking-reward-calculator-1)** to estimate your rewards
+- Remember: rewards depend on total network participation
+
+3. **Withdraw Rewards**
+
+**✅ One-Click Withdrawal!** Co-staking rewards are automatically included when you withdraw BTC staking rewards.
+
+**Easiest way: Click the "Rewards" button in the dashboard!**
+
+<ThemedImage
+  alt="Review and Confirm"
+  sources={{
+    light: useBaseUrl('/img/guides/co-staking/rewards.png'),
+    dark: useBaseUrl('/img/guides/co-staking/rewards_dark.png'),
+  }}
+/>
+
+- **Automatically withdraws**:
+- ✅ BTC staking rewards
+- ✅ Co-staking rewards
+- One click to claim all related rewards!
+
+
+## Important Notes
+
+### Requirements
+
+✅ **Must have active BTC delegations** to Finality Providers
```

Comment:
Please do new lines for each point to make it more readable. 

![Uploading image.png…]()